cmd/sandbox-ce/cmd/install.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/smacker/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type installCmd struct {
+	Command `name:"install" short-description:"Install"`
+}
+
+func (c *installCmd) Execute(args []string) error {
+	err := compose.Run(context.Background(),
+		"run", "--rm", "superset", "./docker-init.sh")
+
+	if err != nil {
+		return err
+	}
+
+	err = compose.Run(context.Background(), "up")
+
```

Comment:
just a nit pick. But usually we don't put empty line in such cases...

Reply:
fixed